README.md

```diff
@@ -31,17 +31,16 @@ See the development roadmap for more details (`vignette("development", package =
 
 ## Installation
 
-Get started with **{laminr}** by installing the development version directly from GitHub:
+Get started with **{laminr}** by installing the package from CRAN:
 
 ```r
-# install.packages("remotes")
-remotes::install_github("laminlabs/laminr")
+install.packages("laminr")
 ```
 
 To include all suggested dependencies for enhanced functionality, use:
 
 ```r
-remotes::install_github("laminlabs/laminr", dependencies = TRUE)
+install.packages("laminr", dependencies = TRUE)
 ```
 
```

Comment:
If you wanted you could add a section about installing the development version from GitHub but not sure it's important

Reply:
True, though the installation instructions are already quite long. It can def be added ! Already merging this so pkgdown is updated